tests/test_class_attributes.rs

```diff
@@ -124,9 +124,9 @@ fn test_fallible_class_attribute() {
                 .getattr("getvalue")?
                 .call0()?
                 .downcast::<PyString>()?
-                .to_str()?
-                .to_owned();
-            let sys = py.import("sys")?;
+                .to_cow()?
+                .into_owned();
```

Comment:
👍 nicely done figuring out the `to_str` / `to_cow` tweaks on `PyString`. I guess we should probably add a note about `PyString::to_str` to the migration guide; I don't think we have one at the moment...

Reply:
Yeah, thats a good idea. I think I stumbled over this twice already...